_pages/dev/event-service.md

```diff
@@ -124,3 +133,60 @@ onClick() {
   this.events.dispatch(new CustomUIEvent(...));
 }
 ```
+
+## Event Type Inheritance
+
+{% capture version_note %}
+{{ site.version_note_part1a }} 3.1 {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+Parent events, also called umbrella events, allow you to group similar events under one common event. The parent event can be an abstract class or a regular class. By subscribing to the parent event, you get emissions from all of the child events that inherit it.
```

Comment:
Maybe we change the wording here a bit, as we don't really want to call them umbrella events. We just used that expression to visually describe what the parent events could be useful for. Maybe we can even remove the term all together.

Reply:
Sounds good 👍